observation_portal/sciapplications/migrations/0006_auto_20210707_0334.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.23 on 2021-07-07 03:34
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sciapplications', '0005_auto_20210628_0304'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='scienceapplication',
+            name='tags',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=255), blank=True, default=list, help_text='List of strings tagging this application', size=None),
```

Comment:
Is this migration just to fix the spelling in help text? We should probably just combine the migrations in the PR so there is just one.

Reply:
Yeah, just a typo. Will combine.